Lib/pstats.py

```diff
@@ -25,9 +25,28 @@
 import time
 import marshal
 import re
+from enum import Enum
 from functools import cmp_to_key
 
-__all__ = ["Stats"]
+__all__ = ["Stats", "SortKey"]
+
+
+class SortKey(str, Enum):
+    CALLS = 'calls'
+    NCALLS = 'ncalls'
+    CUMULATIVE = 'cumulative'
+    CUMTIME = 'cumulative'
+    MODULE = 'module'
+    FILE = 'file'
+    FILENAME = 'filename'
+    LINE = 'line'
+    NAME = 'name'
+    NFL = 'nfl'
+    PCALLS = 'pcalls'
+    STDNAME = 'stdname'
+    TIME = 'time'
+    TOTTIME = 'tottime'
+
```

Comment:
My initial understanding of this was flawed.  Since we need to support the old string values we can't just have CUMTIME and CUMULATIVE map to "cumulative" because then "cumtime" is gone (unless we do extra work, which I see you did).

What we need here is a `MultiValueEnum`, so `SortKey.CUMULATIVE` maps to both "cumulative" _and_ "cumtime".

The Enum should look like this:

    class SortKey(str, Enum):

        CALLS = 'calls', 'ncalls'
        CUMULATIVE = 'cumulative', "cumtime"
        MODULE = 'module', 'filename', 'file'
        LINE = 'line'
        NAME = 'name'
        NFL = 'nfl'
        PCALLS = 'pcalls'
        STDNAME = 'stdname'
        TIME = 'time', 'tottime'

        def __new__(cls, *values):
            obj = object.__new__(cls)
            # first value is canonical value
            obj._value_ = values[0]
            for other_value in values[1:]:
                cls._value2member_map_[other_value] = obj
            obj._all_values = values
            return obj

Operations like `SortKey('file')` will return `SortKey.MODULE`.  The downside is that `SortKey.FILE` is undefined, but I think that is an acceptable trade-off since we are still supporting the original string values, and those values can be used to get a correct `SortKey` member.